frontend/src/login/login.js

```diff
@@ -65,7 +65,7 @@ class loginPage extends Component {
                 (r) => {
                     if(r && r.status && r.status === 200){
                         this.props.login();
-                        this.setState({isPageLoading: false})
+                         this.setState({isPageLoading: false})
```

Comment:
Should this end in a `;`?

Reply:
It can but doesn't have to cos javascript I guess